Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md

```diff
@@ -1,7 +1,7 @@
 # Create resources for the AKS Priavte Cluster
```

Comment:
I think we may need to restructure this page a bit. Especially after some of the feedback we have received from a recent bug bash. We will also need to introduce blue green deployment in some way. Maybe an intro to blue/green and then how to use it, and then our recommendation for using it. We can discuss this on a call.